.github/workflows/deploy-cognito.yml

```
@@ -0,0 +1,61 @@
+name: Deploy Cognito stack to devplatform secure pipeline
+
+on:
+  push:
+    branches:
+      - main
+      - SSE-2450_cognito_deploying_to_build
+    paths:
+      - backend/cognito/**
+      - .github/workflows/**
```

```suggestion
    branches: [ main ]
    paths: [ backend/cognito/** ]
```
Remove test branch, use only cognito path (add manual trigger for workflow changes)